### PR TITLE
refactor: lazy import for markdown package

### PR DESCRIPTION
**Pull request by jirikrepl**
this is breaking change but it saves 9kb in the initial bundle

markdown pacakge is using lazy imports and will not be added to the initial bundle.

to use `{{#markdown}}...{{/markdown}}` import it first with:

`import 'meteor/markdown';`

### CODE REVIEW

**Comment by StorytellerCZ** [comment]
@jirikrepl Thanks! While at it, can you also bump `templating` to `1.4.0` in `package.js` while you are at it? If you want you can also update the readme with how to use it, but do please note that this package is deprecated.

**Comment by jirikrepl** [comment]
Thanks! In the readme.md of this package I put a link to these docs https://docs.meteor.com/packages/markdown.html

And I am proposing this change in the docs, which explain this change
https://github.com/meteor/docs/pull/722

File: packages/deprecated/markdown/README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # markdown
-[Source code of released version](https://github.com/meteor/meteor/tree/master/packages/markdown) | [Source code of development version](https://github.com/meteor/meteor/tree/devel/packages/markdown)
+[Source code of released version](https://github.com/jirikrepl/meteor/tree/master/packages/deprecated/markdown) | [Source code of development version](https://github.com/jirikrepl/meteor/tree/devel/packages/deprecated/markdown)
```
Comment (StorytellerCZ) [inline packages/deprecated/markdown/README.md:2]: This should point to `meteor`, not your personal repo.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

**Comment by jirikrepl** [comment]
ok, thanks. I fixed it

**Review by StorytellerCZ — APPROVED** [review]

**Comment by StorytellerCZ** [comment]
Given this change let's make a major version bump for 2.3 release and then move it to the Blaze repository where this package belongs.

**Pull request merged**